src/dhtrunner.cpp

```diff
@@ -154,21 +155,28 @@ DhtRunner::run(const SockAddr& local4, const SockAddr& local6, const DhtRunner::
             std::cerr << "Can't start peer discovery (IPv6): " << e.what() << std::endl;
         }
     }
+
+    current_node_netid_ = config.dht_config.node_config.network;
```

Comment:
not needed. Build the msgpack object from this function using config.dht_config.node_config.network